src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift

```diff
@@ -254,7 +254,7 @@ extension RiskProvider: RiskProviding {
 			group.leave()
 		}
 
-		guard group.wait(timeout: .now() + .seconds(60)) == .success else {
+		guard group.wait(timeout: .now() + .seconds(60 * 8)) == .success else {
```

Comment:
is any reasoning why 8 min is the perfect amount of time to wait?

Reply:
Its a requirement from the story.